src/components/DialogHeader/DialogHeader.tsx

```diff
@@ -0,0 +1,60 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React, { ReactElement } from 'react';
+import { merge } from '@utilities/merge';
+import { IconCross } from '@foundation/Icon';
+import { Badge, BadgeProps } from '@components/Badge';
+
+export type DialogHeaderProps = {
+    title: string;
+    size?: DialogHeaderSize;
+    collapseBottom?: boolean;
+    close?: boolean;
+    onClose?: () => void;
+    icon?: ReactElement;
+    badge?: BadgeProps;
+};
+
+export enum DialogHeaderSize {
+    Null = 'Null',
```

Comment:
Null?

Reply:
Isn't it better to be `headerSize = 'large' | 'medium' | 'small'`? Otherwise instead of `null` maybe `default`?

Reply:
Changed to Default.